### DISCUSSION TITLE
Existing packages for modelling H5N1 human-to-human transmission

### DISCUSSION THREAD

**Discussion by jamesmbaazam** [epiverse-trace/epiverse-trace.github.io#329] [Q&A]
Following up on #284, what tools are available for modelling H5N1 human-to-human transmission scenarios and how differently would we model/analyse H5N1 compared to mpox given that human-to-human transmission driven outbreaks are still limited? What are some potential questions or common tasks to expect in analysing H5N1?

**Comment by adamkucharski** [comment]
Thanks for starting this discussion - could be a nice follow up blog post with more specific examples? One of the major differences in transmission dynamics is the likely importance network structure for mpox vs age-stratified SIR-like dynamics for respiratory infection.

A few areas of ongoing work with our and other packages:

- [Using {epichains} and H5 cluster sizes in US to estimate R](https://github.com/epiverse-trace/howto/blob/h5_example/analyses/quantify_transmission/reproduction_number_cluster_size.qmd), including a prior based on H5N1 in Indonesia (with prior extracted and defined using {epiparameter}).
- [Using {epichains} to explore potential outbreak dynamics following importation](https://github.com/jamesmbaazam/h5n1_uk_scenario_modelling) (which you're obviously aware of!)
- [Adapting {ringbp} to include wider pathogen scenarios](https://github.com/epiforecasts/ringbp/pull/77). There is also a [specific H5N1 branch](https://github.com/epiforecasts/ringbp/tree/h5n1) with quick edits to customise, but for sustainability this needs to be incorporated into the main PR changes @joshwlambert has suggested. This would be good target for a dev day/sprint once people back from IDM I think.
- Longer-term potential pandemic scenarios (e.g. if becomes more transmissible). {epidemics} would be well placed to handle this, especially as we incorporate more capacity to handle age-specific hospitalisation/fatality risk etc. Now we're [building in {odin} backend](https://github.com/epiverse-trace/epidemics/issues/254) with @bahadzie , will be much easier to add model variants (e.g. with spillover introductions as well as SIR dynamics).
- With @lloydchapman, we've also [been collating some library models](https://github.com/LloydChapman/seir) for {odin.dust} and {mcstate}. I've got an initial script to reproduce estimation of epidemic size from early exported cases for COVID using these models - will push branch shortly.
